core/src/main/java/com/digitalpebble/stormcrawler/protocol/selenium/NavigationFilter.java

```diff
@@ -27,7 +27,7 @@
      * @param stormConf The Storm configuration used for the parsing bolt
      * @param filterParams the filter specific configuration. Never null
      */
-    public void configure(@SuppressWarnings("rawtypes") Map stormConf, JsonNode filterParams) {}
+    public void configure(Map stormConf, JsonNode filterParams) {}
```

Comment:
Why not `Map<String,Object>` here (was changed on other places accordingly) ?

Reply:
see https://github.com/DigitalPebble/storm-crawler/pull/937#discussion_r801502015